packages/manager/apps/web/client/app/hosting/cdn/order/hosting-cdn-order.routing.js

```diff
@@ -96,15 +97,9 @@ export default /* @ngInject */ ($stateProvider) => {
       serviceName: /* @ngInject */ ($transition$) =>
         $transition$.params().productId,
 
-      serviceOption: /* @ngInject */ (
-        goBackWithError,
-        serviceName,
-        $translate,
-        HostingCdnOrderService,
-      ) =>
-        HostingCdnOrderService.getServiceOption(serviceName).catch((error) =>
-          goBackWithError(get(error, 'data.message', error)),
-        ),
+      serviceOption: /* @ngInject */ (availableOptions, goBackWithError) =>
+        find(availableOptions, { family: 'cdn' }) ||
+        goBackWithError('No serviceOption found'),
```

Comment:
no translation?

Reply:
I only moved the code around, I didn't change the behaviour